research/gaia/gremlin/gremlin_core/src/structure/graph.rs

```diff
@@ -23,11 +23,12 @@ pub struct QueryParams<E: Element + Send + Sync> {
     pub limit: Option<usize>,
     pub props: Option<Vec<PropKey>>,
     pub filter: Option<Arc<Filter<E, ElementFilter>>>,
+    pub snapshot_id: Option<i64>,
```

Comment:
It looks like MaxGraph already have a definition of `SnapshotId`., import it instead of directly using `i64`